Update ts-jest to 26.3.0

Right now you get a lot of warnings while running tests, e.g. in `plugin/catalog-backend`:

```
ts-jest[versions] (WARN) Version 4.0.3 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=3.8.0 <4.0.0-0). Please do not report issues in ts-jest if you are using unsupported versions.
```

This could be resolved by updating ts-jest to 26.3.0. But I noticed that this will cause other test failures related to the SVGs:

```
 Test suite failed to run

    ../../packages/core/src/layout/ErrorPage/MicDrop.tsx:19:27 - error TS2307: Cannot find module './mic-drop.svg' or its corresponding type declarations.

    19 import MicDropSvgUrl from './mic-drop.svg';
                                 ~~~~~~~~~~~~~~~~

FAIL src/components/RegisterComponentPage/RegisterComponentPage.test.tsx
  ● Test suite failed to run

    ../../packages/core/src/components/EmptyState/EmptyStateImage.tsx:18:31 - error TS2307: Cannot find module './assets/missingAnnotation.svg' or its corresponding type declarations.

    18 import missingAnnotation from './assets/missingAnnotation.svg';
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ../../packages/core/src/components/EmptyState/EmptyStateImage.tsx:19:27 - error TS2307: Cannot find module './assets/noInformation.svg' or its corresponding type declarations.

    19 import noInformation from './assets/noInformation.svg';
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ../../packages/core/src/components/EmptyState/EmptyStateImage.tsx:20:29 - error TS2307: Cannot find module './assets/createComponent.svg' or its corresponding type declarations.

    20 import createComponent from './assets/createComponent.svg';
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ../../packages/core/src/components/EmptyState/EmptyStateImage.tsx:21:21 - error TS2307: Cannot find module './assets/noBuild.svg' or its corresponding type declarations.

    21 import noBuild from './assets/noBuild.svg';
                           ~~~~~~~~~~~~~~~~~~~~~~
```

--- a/packages/cli/config/jest.js
+++ b/packages/cli/config/jest.js
@@ -77,14 +77,17 @@ async function getConfig() {
       '\\.(css|less|scss|sss|styl)$': require.resolve('jest-css-modules'),
     },
 
+    globals: {
+      'ts-jest': {
+        isolatedModules: true,
+      },
+    },
+
     // We build .esm.js files with plugin:build, so to be able to load these in tests they need to be transformed
     // TODO: jest is working on module support, it's possible that we can remove this in the future
     transform: {
       '\\.esm\\.js$': require.resolve('jest-esm-transformer'),
-      '\\.(js|jsx|ts|tsx)$': [
-        require.resolve('ts-jest'),
-        { isolatedModules: true },
-      ],
+      '\\.(js|jsx|ts|tsx)$': require.resolve('ts-jest'),
       '\\.(bmp|gif|jpg|jpeg|png|frag|xml|svg)$': require.resolve(
         './jestFileTransform.js',
       ),

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -96,7 +96,7 @@
     "sucrase": "^3.16.0",
     "tar": "^6.0.1",
     "terser-webpack-plugin": "^1.4.3",
-    "ts-jest": "^26.0.0",
+    "ts-jest": "^26.4.3",
     "ts-loader": "^7.0.4",
     "typescript": "^4.0.3",
     "url-loader": "^4.1.0",

